_members/cha-jinny

```diff
@@ -0,0 +1,12 @@
+---
+name: Jinny Cha
+image: images/cha-jinny.jpg
+role: phd
+links:
+  email: jzc0159@auburn.edu
+education:
+  - Auburn University, B.S. in Chemical Engineering
+---
+
+Jinny joined the lab in 2023. Her research develops complex confining 
+boundaries for multiparticle coliision dynamics.
```

Comment:
Maybe something like this would be a bit clearer and more narrative?
```suggestion
Jinny joined the lab in 2023. She is developing our open-source multiparticle 
coliision dynamics software in HOOMD-blue to include complex confining
boundaries.
```